src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,60 @@
-export const Tabs = () => {};
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+import { TabInfo } from './TabInfo';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const [selectedTab, setSelectedTab] = useState(
+    tabs.find(tab => tab.id === selectedTabId) || tabs[0],
+  );
```

Comment:
why do you need to create state here?
```suggestion
  const selectedTab =  tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```